### PR TITLE
Various item fixes

### PR DESCRIPTION
**Pull request by SquawkyArctangent**

### CODE REVIEW

**Comment by Trildar** [comment]
What editor do you use anyway? You really ought to get one/set up your current one to automatically handle trimming whitespace on empty lines and ensuring empty line at end of file.

**Comment by SquawkyArctangent** [comment]
notepad++

**Comment by Trildar** [comment]
There's an EditorConfig plugin: https://github.com/editorconfig/editorconfig-notepad-plus-plus

File: game/scripts/vscripts/items/lucience.lua [hunk 1]
```diff
@@ -260,18 +260,17 @@ modifier_item_lucience_regen_effect = class(ModifierBaseClass)
...
-  return {
-    MODIFIER_PROPERTY_HEALTH_REGEN_CONSTANT
-  }
-end
+function modifier_item_lucience_regen_effect:OnIntervalThink()
+  local parent = self:GetParent()
 
-function modifier_item_lucience_regen_effect:GetModifierConstantHealthRegen()
-  return self.regenBonus
+  parent:Heal(self.regenBonus / self.healInterval, self:GetParent())
```
Comment (chrisinajar) [inline game/scripts/vscripts/items/lucience.lua]: shouldn't this either be `self.regenBonus * self.healInterval` or `self.regenBonus / self:GetAbility():GetSpecialValueFor("heals_per_sec")`

File: game/scripts/vscripts/modifiers/modifier_octarine_vampirism_buff.lua [hunk 2]
```diff
@@ -2,103 +2,117 @@ modifier_octarine_vampirism_buff = class(ModifierBaseClass)
...
-  end
-
-  return false
+function modifier_octarine_vampirism_buff:IsHidden()
+	return true
 end
 
 function modifier_octarine_vampirism_buff:IsPurgable()
-  return false
+	return false
```
Comment (chrisinajar) [inline game/scripts/vscripts/modifiers/modifier_octarine_vampirism_buff.lua]: It looks like you accidentally messed up the indentation for the whole rest of this file. If you install that editorconfig plugin and re-save this file it should fix itself.

**Pull request merged**